### ISSUE TITLE
[Feature] Bedrock-style particles.

### ISSUE THREAD

**Issue by seriousfreezing** [Hantonik/FancyBlockParticles#8]
### Feature description:

I realized that to make a style similar to Bedrock's, you could just use the standard 2D particles and rotate them around the player.
Could you tell me if this could be implemented or would there be a problem?

### Minecraft version:

1.21

### Modloader:

Fabric

**Comment by Hantonik** [comment]
Hey
I appreciate your message, but I don't quite understand what you mean. Can you provide more information about your idea and give some examples?

**Comment by seriousfreezing** [comment]
Of course. The particles in the Bedrock version are still 2D, but they are around the camera. So every position or rotation of the camera, they turn towards the camera, so I think it would be interesting to have this both to continue having a particle system still in 2D but improved allowing it to be close to Bedrock;
